### PR TITLE
Workers fixes to support background sending of e-mails.

### PR DESCRIPTION
**Pull request by andriikaplanovskyi**
Workers fixes to support background sending of e-mails (Obelisk out-of-the-box logic).

### CODE REVIEW

File: src/OneBeyond.Studio.Obelisk.Workers/Program.cs [hunk 1]
```diff
@@ -68,6 +76,19 @@ public static void Main(string[] args)
 
         serviceCollection.AddAutoMapper(typeof(Application.AssemblyMark).Assembly);
 
+        serviceCollection.AddTransient<ITemplateRenderer, HandleBarsTemplateRenderer>();
+
+        if (environment.IsDevelopment())
+        {
+            serviceCollection.AddEmailSender(
```
Comment (fabiomaistro) [inline src/OneBeyond.Studio.Obelisk.Workers/Program.cs:83]: Sure this is the right Options, coming from` SmtpEmailSender` namespace?

File: src/OneBeyond.Studio.Obelisk.Workers/Program.cs [hunk 2]
```diff
@@ -68,6 +76,19 @@ public static void Main(string[] args)
...
+                configuration.GetOptions<SmtpEmailSender.Options.EmailSenderOptions>("EmailSender:Folder"));
+        }
+        else
+        {
+            serviceCollection.AddEmailSender(
```
Comment (fabiomaistro) [inline src/OneBeyond.Studio.Obelisk.Workers/Program.cs:88]: Why are we forcing production environment to use sendgrid, what if we want to use some other provider?

**Review by fabiomaistro — CHANGES_REQUESTED** [review]

File: src/OneBeyond.Studio.Obelisk.Workers/Program.cs (hunk 1 above)
Comment (andriikaplanovskyi) [reply src/OneBeyond.Studio.Obelisk.Workers/Program.cs:83]: Good catch. Fixed. The only problem with this catch it was expected to be done approx 3 years ago. Have a look at line 161

https://dev.azure.com/dcslsoftwareltd/DCSL%20Tooling/_git/Waterloo%20Template?path=/WaterlooTemplate.Application.WebApi/Program.cs

File: src/OneBeyond.Studio.Obelisk.Workers/Program.cs (hunk 2 above)
Comment (andriikaplanovskyi) [reply src/OneBeyond.Studio.Obelisk.Workers/Program.cs:88]: @fabiomaistro, we just copy the out-of-the-box logic we provide in WebAPI.

https://github.com/onebeyond/onebeyond-studio-obelisk/blob/main/src/OneBeyond.Studio.Obelisk.WebApi/Program.cs#L146

It's a template, each project will adjust the email configuration to their needs.

**Comment by andriikaplanovskyi** [comment]
@fabiomaistro I pushed the fixes, pls have a look when yo're free.

**Review by fabiomaistro — APPROVED** [review]

**Pull request merged**